app/controllers/concerns/idv_step_concern.rb

```diff
@@ -118,7 +118,7 @@ def url_for_latest_step
     url_for(controller: step_info.controller, action: step_info.action)
   end
 
-  def clear_future_steps!
-    flow_policy.undo_future_steps_from_controller!(controller: self.class)
+  def clear_future_steps!(controller: self.class)
```

Comment:
I would vote for this to be a new method, `clear_future_steps_from!(controller:)` and have `clear_future_steps!` call that with `self.class`

Reply:
Added in 37a127c90928f49768559ac2e2e609aaacd30657